.github/workflows/nightly-test-integrations-1.15.x.yml

```diff
@@ -71,36 +76,34 @@ jobs:
       - name: Generate Envoy Job Matrix
         id: set-matrix
         env:
-          # this is further going to multiplied in envoy-integration tests by the 
-          # other dimensions in the matrix.  Currently TOTAL_RUNNERS would be
-          # 14 based on these values:
-          # envoy-version: ["1.22.11", "1.23.12", "1.24.12", "1.25.11", "1.26.8", "1.27.5", "1.28.3"]
-          # xds-target: ["server", "client"]
-          TOTAL_RUNNERS: 7
+          # TEST_SPLITS sets the number of test case splits to use in the matrix. This will be
+          # further multiplied in envoy-integration tests by the other dimensions in the matrix
+          # to determine the total number of runners used.
+          TEST_SPLITS: 4
```

Comment:
It took me until this PR to realize that (AFAICT this number does not need to be changed after every Envoy version introduction - the comment I believe was simply noting that the total runners is one of 3 factors in the matrix that multiply to get the total effective runner count. I've updated the env var here to `TEST_SPLITS` to try to make that more obvious.